Dataset.py

```diff
@@ -242,13 +245,19 @@ def sliding_window(self, xr):
   # noinspection PyMethodMayBeStatic
   def preprocess(self, seq):
     """
+    TODO: This seems to be only called in a function of CachedDataset, which
+      is then only called in HDFDataset. Seems to be completely useless,
+      as there is no override at any point.
```

Comment:
I don't remember anymore.
Maybe there were some other usages in the past? You could check the history.
But if there are none right now, sure, there is no reason to not remove it.